src/org/omegat/filters2/moodlephp/MoodlePHPFilter.java

```diff
@@ -135,9 +135,11 @@ protected void processFile(BufferedReader inFile, BufferedWriter outFile, Filter
                 processBlock(block.toString(), outFile);
                 block.setLength(0);
             }
-            if (!Character.isWhitespace(c)) { // In the regexp, there could be whitespace between " and >
-                previousChar = c;
+            if (c == quotes && previousChar == 92) {
```

Comment:
Representing the chars as ints is quite cryptic; I would suggest changing them to char literals: `'\\'` here, and `'\''` above. In fact there doesn't seem to be a point in having them be `int` at all; you might as well make them `char`.

While you're at it, you might make the `quotes` variable a static final constant, or at least make the local variable `final` to indicate that it's not meant to be modified by the loop.

Reply:
I did something, but I am not able to do everything you suggest.